src/main/java/seedu/staffsnap/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.staffsnap.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.staffsnap.logic.parser.CliSyntax.PREFIX_DESCRIPTOR;
+import static seedu.staffsnap.model.Model.PREDICATE_HIDE_ALL_APPLICANTS;
+import static seedu.staffsnap.model.Model.PREDICATE_SHOW_ALL_APPLICANTS;
+
+import seedu.staffsnap.logic.commands.exceptions.CommandException;
+import seedu.staffsnap.model.Model;
+import seedu.staffsnap.model.applicant.Descriptor;
+
+
+/**
+ * Sorts all Applicants in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorted all Applicants. "
+            + "Parameters: "
+            + PREFIX_DESCRIPTOR + "DESCRIPTOR ";
+    public static final String MESSAGE_SUCCESS = "Sorted all Applicants";
+
+    private final Descriptor descriptor;
+
+    /**
+     * Creates a SortCommand to add the specified {@code descriptor}
+     */
+    public SortCommand(Descriptor descriptor) {
+        requireNonNull(descriptor);
+        this.descriptor = descriptor;
+    }
+
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        model.updateSortedApplicantList(descriptor);
+        model.updateFilteredApplicantList(PREDICATE_HIDE_ALL_APPLICANTS);
+        model.updateFilteredApplicantList(PREDICATE_SHOW_ALL_APPLICANTS);
```

Comment:
This looks like a clever way update the list 🚀 

Not sure if there'll be any side-effects to this down the road, perhaps we can add in a comment to explain the purpose of this workaround for now. 

Down the line when we implement the filter() function, we might encounter a similar issue and have to revisit this.

Reply:
Nice catch Craig, forgot that this was unintuitive.

Updated it with a comment describing why it does this, and potential fix in the future when filter has to be implemented.